src/search_algo.hpp

```diff
@@ -425,6 +434,7 @@ inline void search_impl(LocalDataHolder<TGlobalHolder> & lH, TSeed && seed)
 {
     if constexpr (TGlobalHolder::c_dbIndexType == DbIndexType::FM_INDEX)
     {
+        //TODONE: I don't know whether we want to follow up on this with Simon?
```

Comment:
I think we should try to minimize the stuff we still want to add and fixing the taxonomy sounds more burning than that.